front/app/containers/SiteMap/index.tsx

```diff
@@ -154,216 +155,249 @@ const SiteMap = ({ projects, authUser }: Props) => {
-                  navBarItems
-                    .filter(
-                      (item) => item.relationships.static_page.data === null
-                    )
-                    .map((item) => (
-                      <li key={item.id}>
-                        <Link to={DEFAULT_PAGE_SLUGS[item.attributes.code]}>
-                          {localize(item.attributes.title_multiloc)}
-                        </Link>
+                    {!isNilOrError(projects) && (
+                      <li>
+                        <NavItem
+                          onMouseDown={removeFocusAfterMouseClick}
+                          onClick={scrollTo(projectsSection)}
+                        >
+                          <FormattedMessage {...messages.projectsSection} />
+                        </NavItem>
+                        {hasProjectSubsection && (
+                          <ProjectsSubsectionUl>
+                            {currentSection.current && (
+                              <li>
+                                <NavItem
+                                  onMouseDown={removeFocusAfterMouseClick}
+                                  onClick={scrollTo(currentSection)}
+                                >
+                                  <FormattedMessage
+                                    {...messages.projectsCurrent}
+                                  />
+                                </NavItem>
+                              </li>
+                            )}
+                            {archivedSection.current && (
+                              <li>
+                                <NavItem
+                                  onMouseDown={removeFocusAfterMouseClick}
+                                  onClick={scrollTo(archivedSection)}
+                                >
+                                  <FormattedMessage
+                                    {...messages.projectsArchived}
+                                  />
+                                </NavItem>
+                              </li>
+                            )}
+                            {draftSection.current && (
+                              <li>
+                                <NavItem
+                                  onMouseDown={removeFocusAfterMouseClick}
+                                  onClick={scrollTo(draftSection)}
+                                >
+                                  <FormattedMessage
+                                    {...messages.projectsDraft}
+                                  />
+                                </NavItem>
+                              </li>
+                            )}
+                          </ProjectsSubsectionUl>
+                        )}
                       </li>
-                    ))}
-                {/* Non-custom static pages */}
-                {!isNilOrError(pages) &&
-                  pages
-                    .filter((page) => page.attributes.code !== 'custom')
```

Comment:
Before it only took out the custom pages, now also the proposals page if the feature is not enabled.